doc-content/enterprise-only/getting-started-kogito/proc-kogito-migrate-drl-implement-ruleunits.adoc

```diff
@@ -0,0 +1,152 @@
+[id="proc-kogito-migrate-drl-implement-ruleunits_{context}"]
+= Implementing rule units and automatic REST endpoint generation
+
+[role="_abstract"]
+After migrating rule units to a {KOGITO} microservice, you can implement the rule units and automatic generation of the REST endpoint.
+
+In {KOGITO}, a rule unit contains a set of rules and the facts, against which the rules are matched. Rule units in {KOGITO} also come with data sources. A rule unit data source is a source of the data processed by a given rule unit and represents the entry point, which is used to evaluate the rule unit. There are two types of data sources:
+
+* `DataStream`: This is an appended data source. In `DataStream`, subscribers receive new and past messages, stream can be hot or cold in the reactive streams. Also, `DataStream` cannot be updated or removed.
+* `DataStore`: This data source is for modifiable data, in which subscribers can access the data store by using the added fact.
```

Comment:
Also rewrite this as it follows

`DataStore: This data source is for modifiable data. It is possible to update or remove an object using the FactHandle returned when the object is added into the DataStore.`

Reply:
Done